docker-compose.yml

```diff
@@ -377,6 +386,20 @@ services:
       test: [ "CMD-SHELL", "pg_isready || false" ]
       interval: 30s
       timeout: 3s
+  batch-manager:
+    restart: always
+    profiles: ["production", "testing", "development"]
+    networks:
+      - internal
+    image: ghcr.io/d-fine/dataland/dataland_batch_manager:${DATALAND_BATCH_MANAGER_VERSION}
+    environment:
+      GET_ALL_GLEIF_COMPANIES_FLAG_FILE: /config/perform_full_golden_copy_download_flag
+      INTERNAL_BACKEND_URL: ${INTERNAL_BACKEND_URL:-http://backend:8080/api}
+      DATALAND_BATCH_MANAGER_CLIENT_SECRET: ${DATALAND_BATCH_MANAGER_CLIENT_SECRET}
+    volumes:
+      - ./dataland-batch-manager/config:/config
```

Comment:
Zwei Fragen:
1.) Der Mechanismus ist also, dass wenn dieses flag-file (/config/perform_full_golden_copy_download_flag) existiert (und der Inhalt des flag-files ist an sich egal), man sich die alle GLEIF companies holt (golden copy)?
2.) Und wozu ist der volume-mount ?

Reply:
1. Jo
2. Um das Flag File in den Container zu bekommen. Das wird wahrscheinlich hier so gelöst und nicht im dockerfile, damit  die Company Ingestion nur in einer Umgebung passiert, wo die File bewusst erzeugt wird.

Reply:
zu 2.)  Ok aber da das doch ohnehin nur beim deployen passieren soll, könnte man das doch auch spezifisch in den CD-Skripten kopieren, oder?
Im docker-compose macht es nämlich den Eindruck, als könne/solle man die golden copy auch lokal sich holen können.
Oder was meinst du?

Reply:
Hat sich am Telefon geklärt 👍